wcfsetup/install/files/acp/update_com.woltlab.wcf_5.3_style.php

```diff
@@ -0,0 +1,132 @@
+<?php
+require('global.php');
+
+use wcf\data\style\StyleEditor;
+use wcf\data\style\StyleList;
+use wcf\util\FileUtil;
+
+/**
+ * @author	Tim Duesterhus
+ * @copyright	2001-2020 WoltLab GmbH
+ * @license	GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package	WoltLabSuite\Core
+ */
+
+$styleList = new StyleList();
+$styleList->readObjects();
+
+foreach ($styleList as $style) {
+	$style->loadVariables();
+	$variables = $style->getVariables();
+	$styleEditor = new StyleEditor($style);
+	
+	
+	// 1) Move existing asset path folder out of the way.
+	// It's unlikely that one exists, but having an existing folder will create a small mess.
+	if (file_exists($style->getAssetPath())) {
+		rename($style->getAssetPath(), FileUtil::removeTrailingSlash($style->getAssetPath()) . '.old53/');
+	}
+	
+	// 2) Create asset folder.
+	FileUtil::makePath($style->getAssetPath());
+	
+	// 3) If the file had a custom image folder we need to copy all files into the asset folder.
+	// Moving the files is unsafe, because multiple styles can share a single image folder.
+	if ($style->imagePath != 'images/') {
+		$srcPath = FileUtil::addTrailingSlash(WCF_DIR.$style->imagePath);
+		if ($srcPath == $style->getAssetPath()) {
+			$srcPath = FileUtil::removeTrailingSlash($style->getAssetPath()) . '.old53/';
+		}
+		
+		$iterator = new \RecursiveIteratorIterator(
+			new \RecursiveDirectoryIterator(
+				$srcPath,
+				\FilesystemIterator::SKIP_DOTS
+			), 
+			\RecursiveIteratorIterator::SELF_FIRST
+		);
+		foreach ($iterator as $file) {
+			/** @var \SplFileInfo $file */
+			if (!$file->isFile()) continue;
+			
+			$relative = FileUtil::getRelativePath($srcPath, $file->getPath());
+			FileUtil::makePath($style->getAssetPath() . $relative);
+			copy($file->getPathname(), $style->getAssetPath() . $relative . $file->getBasename());
+		}
+	}
+	$styleEditor->update([
+		'imagePath' => 'images/style-' . $style->styleID . '/',
```

Comment:
```suggestion
		'imagePath' => FileUtil::unifyDirSeparator(FileUtil::getRelativePath(WCF_DIR, $style->getAssetPath())),
```

Rationale: Not hardwiring the asset path here allows us to change it before releasing the update without introducing hidden errors.